### PR TITLE
CB-18-tools-color

### PR DESCRIPTION
**Pull request by sunghyun96**
* **Please check if the PR fulfills these requirements**
- [x] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
The tools component background and text was changed to be consistent with the new color scheme that was chosen. The logo images were also changed to be grayscale to reduce the color clashing. The images will now only show up as color images when the user's mouse hovers over the image.

* **What is the current behavior? What does the code do before you made the new changes?** 
The current behavior was that the tools component was a default configuration of the reactstrap jumbotron. The layout of the jumbotron was the same.

* **What is the new behavior?**
The behavior is that the color scheme has changed for the jumbotron background, text color, and image colors.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
No; there is no breaking change.

* **Other information**:
image

### CODE REVIEW

**Review by Stacey-Masi — APPROVED** [review]
Looks great! I really like how the logo colors appear when hovered over

File: frontend/src/components/LandingPage/Tools/Tools.js [hunk 1]
```diff
@@ -7,7 +7,8 @@ import {
   function Tools() {
     return (
       <div>
-        <Jumbotron fluid>
+        <Jumbotron fluid style={{ background: `linear-gradient(25deg, #e3c3b2 25%, rgba(0, 0, 0, 0) 25%),
```
Comment (Codubee) [inline frontend/src/components/LandingPage/Tools/Tools.js]: What do you think about moving this to the stylesheet? If its not applying to the jumbotron I can show you how.

**Review by Codubee — CHANGES_REQUESTED** [review]
Great work Sung! Colors look great. Just left one comment.

File: frontend/src/components/LandingPage/Tools/Tools.js (hunk 1 above)
Comment (sunghyun96) [reply frontend/src/components/LandingPage/Tools/Tools.js]: Yeah, I was not able to style it in the stylesheet when I was doing it, but I found a work around by using !important to override the default background. Let me know if this was a good/bad way to override it!

**Review by Codubee — APPROVED** [review]
LGTM!

**Pull request merged**